### ISSUE TITLE
Vue 3 menu deselects when using parameter in route

### ISSUE THREAD

**Issue by hieuncong** [yaminncco/vue-sidebar-menu#200]
Hi, Im facing the same issue as [#176](https://github.com/yaminncco/vue-sidebar-menu/issues/176).
But Im currently working on Vue 3 with version 5.0.1 on the next branch and I cannot find `alias` or `exactPath` to achieve the result in the issue above. 
So I wonder is there any way to do this on Vue 3.

**Comment by yaminncco** [comment]
can you show me how menu and routes structure? and yes exactPath and alias have been removed due to the active behavior changes in vue-router v4

**Comment by hieuncong** [comment]
This is in my router
```
      {
        path: 'events/:event_params?',
        name: 'Events',
        component: Events,
        meta: {
          requiresAuth: true
        }
      }
```
And this is in my menu
```
        {
          href: { name: "Events", params: { event_params: "" } },
          title: "Events",
          icon: {
            element: "img",
            attributes: { src: nav2Icon },
          },
        },
```
Everything work fine until I put something different to `""` in the parameter of the URL. Do you have any idea to fix this on Vue 3? Thank you

**Comment by hieuncong** [comment]
Hi again, so is there any way to fix this problem on Vue 3. Please let me know if there is any

**Comment by yaminncco** [comment]
i am working on it

**Comment by hieuncong** [comment]
Thank you yaminncco

**Comment by yaminncco** [comment]
fixed in v5.1.0
